### DISCUSSION TITLE
how can we be better with aqua?

### DISCUSSION THREAD

**Discussion by jdx** [jdx/mise#3896] [General]
I was disappointed to read this: https://github.com/aquaproj/aqua-registry/issues/30430

@suzuki-shunsuke is absolutely correct about everything of course. He should not be dealing with mise support whatsoever. It's up to mise to be compatible with the aqua registry—not the other way around.

Does anyone have any thoughts about what we could do in order to improve this? I don't think it would help for me to state the obvious that you need to come _here_ if you're having issues with mise and that if you post something about aqua you need to verify it's an issue with aqua and not mise. I think anyone reading this post likely already is aware of that.

One idea I had was to fork the aqua repository into a mise repository that follows aqua so at least people could post there instead of directly on aqua. That way I could point the docs to the mise aqua registry instead. It would be largely the same otherwise, I'd just bump it every release or on a cron with all the latest aqua changes. I'm not sure how much that would help though since users may still just go to aquaproj/aqua-registry.

**Comment by suzuki-shunsuke** [comment]
Thank you for opening this discussion to address this problem!

First and foremost, I deeply appreciate the contributions to aqua-registry from mise users.
They have added new tools to aqua-registry and fixed issues.
I believe we can collaborate well.

In general, I think most mise users handle things appropriately.
While there are occasional instances of undesired actions, such situations are relatively rare and, realistically, not entirely avoidable.
We can accept it because this is not so many problem for now.

To streamline the handling of such cases, I’ve created https://github.com/aquaproj/aqua-registry/issues/30430 . Moving forward, I can simply direct individuals to it, avoiding the need to repeat explanations every time.

I don't hope you fork aqua-registry.
This isn't good for both mise and aqua.

Thank you.
